pypeit/specobjs.py

```diff
@@ -348,6 +360,7 @@ def __getitem__(self, item):
             #sobjs_new = np.array(self.specobjs,dtype=object)
             return SpecObjs(specobjs=self.specobjs[item])
 
+    # TODO PROFX this code fails for assignments of this nature sobjs[:].attribute = np.array(5)
```

Comment:
i'm not surprised.  That would be a rather challenging
assignment to code.

Reply:
Yeah I think it can be done. So far making the sobjs a class rather than an astropy table has only  resulted in bugs and pain.